README.md

```diff
@@ -0,0 +1,4 @@
+Cmake assignment
+
+Write cmake file to build and link two files (c++) and one header file
```

Comment:
What you wrote is not a readme. in README you must write
1. What is its purpose.
2. Dependencies (if exists)
3. How to build
4. How to run
5. How to run tests (if exists)
6. etc.

Reply:
+

Reply:
It is still bad. 
1. You wrote the given Task as a project description instead of writing what it does.
2. What bash scripts you wrote here about???

Reply:
++